src/main/java/com/extendedclip/papi/expansion/server/ServerExpansion.java

```diff
@@ -157,11 +158,23 @@ public String onRequest(OfflinePlayer p, @NotNull String identifier) {
 				long seconds = TimeUnit.MILLISECONDS.toSeconds(ManagementFactory.getRuntimeMXBean().getUptime());
 				return formatTime(Duration.of(seconds, ChronoUnit.SECONDS));
 			case "total_chunks":
-				return String.valueOf(cache.get("chunks", k -> getChunks()));
+				try {
+					return String.valueOf(cache.get("chunks", this::getChunks));
+				} catch (ExecutionException e) {
+					throw new RuntimeException(e);
```

Comment:
Is it really worth to do this?
I feel like just returning null here is better as this could easily spam the entire console to death.

Also, maybe consider a convenience method to access the cache?
Like...
```java
private String getCached(String key, Callable<? extends V> call) {
    try {
        return String.valueOf(cache.get(key, callable));
    } catch (ExecutionException e) {
        return null;
    }
}
```

You should then be able to just have:
```java
switch (identifier) {
    // ...
    case "total_chunks":
        return getCached("chunks", this::getChunks);
    case "total_living_entities":
        return getCached("livingEntities", this::getLivingEntities);
    case "total_entities":
        return getCached("totalEntities", this::getTotalEntities);
    // ...
}
```

This would IMO clean up the code a bit.